bika/lims/browser/fields/aranalysesfield.py

```diff
@@ -111,10 +113,14 @@ def set(self, instance, service_uids, prices=None, specs=None, **kwargs):
             value = dictionary: defined in ResultsRange field definition
 
         """
-        if not service_uids:
+        if not items:
             return
 
-        assert type(service_uids) in (list, tuple)
+        assert type(items) in (list, tuple)
```

Comment:
Better use `assert isinstance(items, (list, tuple)), "items must be a list or a tuple"`